src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String SEPARATOR_IN_LINE = ", ";
+    private static final StringBuilder report = new StringBuilder();
```

Comment:
The report StringBuilder should not be static. Make it a local variable inside the generateReport method.